tests/perfalert/test_analyze.py

```diff
@@ -29,21 +33,18 @@ def test_calc_t(self):
         self.assertEqual(calc_t([0.0, 0.0], [1.0, 1.0]), float('inf'))
 
 
-class TestTalosAnalyzer(unittest.TestCase):
+class TestAnalyzer(unittest.TestCase):
+
+    def test_analyze_t(self):
+        a = Analyzer()
 
-    def get_data(self):
         times = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15]
         values = [0, 0, 0, 0, 0, 0, 0, 0, 1, 1,  1,  1,  1,  1,  1,  1]
-        return [PerfDatum(t, float(v)) for t, v in zip(times, values)]
+        for (t, v) in zip(times, values):
+            a.add_data(t, v)
```

Comment:
I don't think it's a *bad* idea, since most of the operations we run against these numbers will assume floating point.

Note that this is just preserving the existing behavior, we're not adding new code here.